wordpress-mobile/WordPress-Android
# [Gutenberg] Error Notice Improvements - TESTING ONLY

To test:

https://github.com/WordPress/gutenberg/pull/30273

For the Audio block, now the error notice is shown when an upload failure takes place. 

With the WP Android PR. 

1. (Audio block is enabled on free for testing.)
2. Add an audio block. 
3. Upload any file.
4. Notice the failure. 

Light Mode | Dark Mode 
--------|-------
<kbd><img src="https://user-images.githubusercontent.com/1509205/112577583-9bae5f00-8dc2-11eb-8d09-19064f52d007.png" width="320"></kbd>| <kbd><img src="https://user-images.githubusercontent.com/1509205/112577586-9cdf8c00-8dc2-11eb-8d03-3453ff93b78f.png" width="320"></kbd>       

### Media & Text

For this block the `onUploadError` that was set to the `onError` of the `MediaPlaceholder` does not work on mobile. I had to put a manual `onUploadError` here to test
https://github.com/WordPress/gutenberg/blob/rnmobile/error-notice-fixes/packages/block-library/src/media-text/media-container.native.js#L161

I removed the notice, but I wasn't sure if what message we would want to show or if we should just eliminate notices from here?

Light Mode | Dark Mode 
--------|-------
<kbd><img src="https://user-images.githubusercontent.com/1509205/112577809-1e371e80-8dc3-11eb-8ef8-463609f08c16.png" width="320"></kbd> |      <kbd><img src="https://user-images.githubusercontent.com/1509205/112577852-3149ee80-8dc3-11eb-9477-a3d4699db9ac.png" width="320"></kbd>
PR submission checklist:
